benchmarks/precision/precision.py

```
@@ -0,0 +1,587 @@
+    line: int
+    column: int
+    snippet: str
+    severity: str
+    description: str
+    duplicate_index: int
+    label: str | None = None
+    justification: str | None = None
+
+
+def run_cmd(args: list[str], cwd: Path | None = None, allow_findings: bool = False) -> str:
+    proc = subprocess.run(args, cwd=cwd, text=True, capture_output=True)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)